tensorflow/python/kernel_tests/concat_op_test.py

```diff
@@ -714,6 +715,26 @@ def testNegativeDim(self):
       ans = self.evaluate(off)
       self.assertAllEqual(ans, [[0, 0, 0], [2, 0, 0], [3, 0, 0]])
 
+  def testCreateMemDecBlockedFormat(self):
+    """Try to create the mkl concat operation
+    when one of the input's memory descriptor is in blocked format"""
+    if test_util.IsMklEnabled():
+      s0 = np.ones((1, 8188, 4092, 1), dtype=np.uint8).astype(np.float32)
+      s1 = array_ops.strided_slice(s0, [0, 1, 1, 0], [0, -1, -1, 0],
+                                   [1, 1, 1, 1], begin_mask=9, end_mask=9)
+      s2 = array_ops.slice(s1, [0, 0, 0, 0], [-1, -1, -1, 1])
+      s3_1 = array_ops.slice(s2, [0, 4, 4, 0], [-1, 8178, 4082, 1])
+      s3_2 = array_ops.slice(s2, [0, 4, 4, 0], [-1, 8178, 4082, 1])
+      filter4_1 = constant_op.constant([[[[1.18, -0.51]]]])
+      s4_1 = nn_ops.conv2d(s3_1, filter4_1,
+                           strides=[1, 1, 1, 1], padding="VALID")
+      filter4_2 = constant_op.constant([[[[1.38, -0.11]]]])
+      s4_2 = nn_ops.conv2d(s3_2, filter4_2,
+                           strides=[1, 1, 1, 1], padding="VALID")
+      s5_1 = array_ops.slice(s4_1, [0, 6, 6, 0], [-1, 1, 1, -1])
+      s5_2 = array_ops.slice(s4_2, [0, 6, 6, 0], [-1, 1, 1, -1])
+      x_concat = array_ops.concat([s5_1, s5_2], 3)
+      self.evaluate(x_concat)
```

Comment:
Could you please add a check for correctness as well, e.g., `self.assertAllEqual(self.evaluate(x_concat), solution)`? Otherwise, please add a comment that this test is only meant to check that there isn't a crash.